### ISSUE TITLE
Translation to other languages

### ISSUE THREAD

**Issue by kevin-w-du** [seed-labs/seed-labs#3]
To help bring the SEED labs to non-English speaking countries, help is needed to translate the SEED labs into other languages. The entire translation task is huge, but if we can get more people to work together, it is quite doable. We will create a separate github repository for each language (the Chinese translation is currently under way).

**Comment by lcjury** [comment]
I would be happy to help translating some files to Spanish, I'm interested on the web category :)

**Comment by kevin-w-du** [comment]
That's great. I will create a repo and then invite you to join. Together, we can try to get more people to join us in this translation project. Thanks for willing to help.

**Comment by Unviray** [comment]
I would be happy to help translating to **French** :)

**Comment by kevin-w-du** [comment]
Thanks for the helping. I have just created a seedlabs-french repo, and have invited you to join.

**Comment by Unviray** [comment]
Which file we translate ?

- README
- ...

**Comment by kevin-w-du** [comment]
You will have to go to the seed-labs repo, find the .tex files in the category-*** folders. The sources files of all the lab descriptions are stored there, and they are the main focus of the translation. I didn't copy those files to the seedlabs-french repo.

**Comment by mrtlgz** [comment]
I can help translating to Turkish

**Comment by kevin-w-du** [comment]
That's great. I will create a new repo for this translation project, and then invite you to join this repo. Thanks for the help.

**Comment by rubenoca** [comment]
If you need an additional set of eyes/help with translating to Spanish, please let me know! I am more than willing to help.

**Comment by kevin-w-du** [comment]
Yes, the more the better. I have already created a repo for the Spanish translation two months ago (you can find it in `https://github.com/seed-labs`). Right now, there is no activity in that repo. If you can get it started, that will be great. Please make sure to translate the recent version (SEED Labs 2.0).  Before you translate any lab, create an issue in that repo, so others won't duplicate your work. Thanks for the help.

**Comment by francoataffarel** [comment]
Hello Sir, i can help to translate to Brazilian Portuguese.

**Comment by kevin-w-du** [comment]
Thanks. We have a separate repo for that. You can add your translations to that repo: https://github.com/seed-labs/seedlabs-bp
